client/elasticsearch/v2/client.go

```diff
@@ -99,6 +99,10 @@ func (cli *Client) DeleteIndices(ctx context.Context, indices []string) error {
 	return cli.DeleteIndex(ctx, indices[0])
 }
 
+func (cli *Client) CountIndices(ctx context.Context, indices []string) ([]byte, error) {
```

Comment:
add go doc please

Reply:
Please note ,this is for legacy client where this api is not supported, so that is why no comment is added